chore: add note about how to attach debugger (#6273)
netlify/cli: CONTRIBUTING.md

DEBUG=true ./bin/run.js [command]
```

### Attaching a Debugger

When debugging a project, it's super helpful to attach a debugger to the CLI. If you use VS Code, here's how you can do it:

1. Open this repository in VS Code.

> 2. Open a "Javascript Debug Terminal" (e.g. by searching for it in the  Command Pallette (CMD+P)). Every Node process that's opened in this terminal will have a debugger attached.

Reviewer: ```suggestion
2. Open a "Javascript Debug Terminal" (e.g. by searching for it in the Command Palette (Shift-Cmd+P)). Every Node process that's opened in this terminal will have a debugger attached.
```

> 2. Open a "Javascript Debug Terminal" (e.g. by searching for it in the Command Palette (Shift-Cmd+P)). Every Node process that's opened in this terminal will have a debugger attached.

Reviewer 2: ```suggestion
2. Open a "JavaScript Debug Terminal" (e.g. by searching for it in the Command Palette (Shift-Cmd+P)). Every Node process that's opened in this terminal will have a debugger attached.
```

> 3. Place a breakpoint somewhere in the CLI. You will have to place them in the compiled `.js` files as opposed to the `.ts` files.

Reviewer 3: Can't we output source maps to fix this? Or is there some other limitation

Author: Not sure if we can, just wanted to document what's working for me.

> 3. Place a breakpoint somewhere in the CLI. You might have to place them in the compiled `.js` files as opposed to the `.ts` files.

Author: Currently, no they will never work. If we enabled source maps though, it could! Let's clarify this for now.
```suggestion
3. Place a breakpoint somewhere in the CLI. You will have to place them in the compiled `.js` files as opposed to the `.ts` files.
```

4. In your JavaScript Debug Terminal, navigate to the project you'd like to debug.

> 5. Run `/path/to/netlify/cli/bin/run.js`. You should see the debugger connecting automatically.

Author: ```suggestion
5. Run `/path/to/netlify/cli/bin/run.js`. The debugger should be connecting automatically.
```
